pkg/virt-handler/vm.go

```diff
@@ -98,6 +98,14 @@ func (d *VMHandlerDispatch) updateVMStatus(vm *v1.VM, cfg *api.DomainSpec) error
 		return err
 	}
 	vm = obj.(*v1.VM)
+
+	// XXX When we start supporting hotplug, this needs to be altered.
+	// Check if the VM is already marked as running. If yes, don't update the VM.
+	// Otherwise we end up in endless controller requeues.
```

Comment:
I would rather say, that depending on what we will have in the status, virt-handler will have to update the VM status. So it is related, but I would not say that it helps virt-handler, since it is the task of virt-handler to fill the status then.